cart.html

```diff
@@ -0,0 +1,167 @@
+            <div class="mr-auto page-link-wrapper font3">
+                <ul class="style-list-none">
+                    <li class="inline-block"><a href="index.html" class="page-link decor-none">Home</a></li>
+                    <li class="inline-block"><a href="shop.html" class="page-link decor-none">Shop</a></li>
+                </ul>
+                
+                
+            </div>
+            <div class="badge-container">
+                <i class="fa fa-search fa-lg" id="toggle-search"></i>
+                <i class="fa fa-user fa-lg" id="user-icon"></i>
+                
+                <a href="wishlist.html" class="avatar avatar-badge text-primary reset"><i class="fas fa-lg fa-heart cart "> </i>
+                    <div class="status-circle icon-top-badge-sm ">4</div>  
+                </a>
+                <a href="cart.html" class="avatar avatar-badge text-primary reset"><i class="fas fa-lg fa-shopping-cart cart "> </i>
+                    <div class="status-circle icon-top-badge-sm ">4</div>  
+                </a>
+              </div>
+            </div>
+        </Header>
+        <aside class="side-section" id="side-section">
+            <div class="navbar-links">
+                <div class="sidebar-head">
+                    
+                    <i class="fa fa-times" id="sidebar-close"></i>
+    
+                </div>
+                <div class="avatar centered mb-lg">
+                    <img class="border-round img-border md-img" src="./assets/toy1.jpg" alt="Avatar">
+                </div><hr/>
+                <ul class="navbar-items">
+                    <li><a class="decor-none" href="index.html"> Home</a></li>
+                    <li><a href="shop.html" class=" quick-link decor-none">Shop now</a></li>
+                    <li><a href="#" class=" quick-link decor-none">Profile</a></li>
+                    <li><a href="#" class=" quick-link decor-none">Orders</a></li>
+                    <li><a href="#" class=" quick-link decor-none">Address</a></li>
+                    <li><a href="#" class=" quick-link decor-none">Settings</a></li>
+                </ul>
+            </div>
+        </aside>
+       
+        <main> <div class="search_input_wrapper" id="container_input_wrapper">
+            <input type="text" class="search_input font2" placeholder="SEARCH....">
+            <i class="fa fa-times fa-2x " id="search-close" ></i>
+        </div>
+        <div class="cart-left-pane">
+           <div class="cart-items">
+               <img src="./assets/item2.jpg" alt="product">
+               <div class="cart-product-detail">
+                   <h3 class="text-vibrant">Wooden Basket</h3>
+                   <p class="text-secondary ">Environment Friendly Baskets</p>
+                   <p class="item-price font3 mb-lg">₹499 only</p>
+                   <div class="quantity mb-lg">
+                       <button>-</button>
+                       <input type="text" disabled value="2">
```

Comment:
The `quantity` div items are not aligned properly. Display them as flex and align them. Or match the height of the buttons and input field

Reply:
Done!